speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/ConfigEnums.h

```diff
@@ -19,6 +19,9 @@ typedef struct Indicator
 /// @brief The different interventions that can be done
 typedef unsigned int InterventionType;
 
+/// @brief The different types of data compression that can be done
+typedef unsigned int DataCompressionType;
```

Comment:
Move typedef and add enums.
- Don't add this typdef in between `InterventionType`  and it's ENUM values.
- Add ENUM values COMPRESSION_NONE, etc